media.go

```diff
@@ -484,6 +485,12 @@ func (item *Item) PreviewComments() []Comment {
 	return nil
 }
 
+// StoryCloseFriends returns a bool
+// If the returned value is true the story was published only for close friends
+func (item *Item) StoryCloseFriends() bool {
```

Comment:
Please change `StoryCloseFriends` to `StoryIsCloseFriends` or something contains `is`.
`IsCloseFriends()`, etc ...